models/groups_test.go

```diff
@@ -534,12 +534,12 @@ func TestGroup_CreateUpdateGroupCleanGroupDescription(t *testing.T) {
 		)
```

Comment:
The name of this test function still references `description` rather than `name`:

Current:
```
func TestGroup_CreateUpdateGroupCleanGroupDescription(t *testing.T) {
```
Suggested:
```
func TestGroup_CreateUpdateGroupCleanGroupName(t *testing.T) {
```